api/package.json

```
@@ -9,6 +9,7 @@
   "scripts": {
     "start": "node bin/spark.js",
     "lint": "standard",
+    "start:fastify": "node bin/spark-fastify.js",
```

```suggestion
    "start:fastify": "node bin/spark-fastify.js",
```

That file doesn't exist